pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/ReplicatedSubscriptionsController.java

```diff
@@ -75,6 +76,10 @@ public class ReplicatedSubscriptionsController implements AutoCloseable, Topic.P
                     "Counter of currently pending snapshots")
             .register();
 
+    private static final Counter timedoutSnapshotsMetric = Counter
```

Comment:
I don't understand how this helps you understand the severity of the issue? It's basically serves a signal that something bad happened. Don't you need to know the backlog size, or am missing something on how this snapshotting process works?